### PR TITLE
Edit permission part of the Role management feature is implemented to the B2B sample application

### PR DESCRIPTION
**Pull request by Achintha444**
## Purpose
Edit permission part of the role management feature is integrated into the B2B sample application. UI of this interface is given below.

Screenshot 2022-11-15 at 09 40 40

### CODE REVIEW

**Review by thivi — APPROVED** [review]

**Comment by lashinijay** [comment]
Shall we mention the related git issue here?

File: b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/permission.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    const setInitialPermissions = useCallback(async () => {
+        setSelectedPermissions(roleDetails.permissions);
+    }, [ roleDetails ]);
+
+    useEffect(() => {
+        setInitialPermissions();
+    }, [ setInitialPermissions ]);
+
+    const onDataSubmit = (response, form) => {
+        if (response) {
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/permission.js:54]: make the code consistent by leaving a new line after declarion.

**Comment by Achintha444** [comment]
> Shall we mention the related git issue here?

Added to descritption

File: b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/permission.js (hunk 1 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/permission.js:54]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/05bcb70be881109f31c421be63d5d5d884094b42

**Pull request merged**